src/audio_unit/mod.rs

```diff
@@ -11,12 +12,17 @@ pub use split::Split;
 pub use transparent::Transparent;
 
 use crate::Result;
+use wmidi::MidiMessage;
 
 pub type Boxed = Box<dyn AudioUnit>;
 
 pub trait AudioUnit: Send + Sync {
     fn process(&mut self, input: &[f32], output: &mut [f32]) -> Result<()>;
 
+    fn handle_midi_messages(&mut self, _midi_messages: &[MidiMessage<'static>]) -> Result<()> {
+        Ok(())
+    }
```

Comment:
Added a default no-op implementation for units that don't care about MIDI messages.